app/helpers/layout_helper.rb

```diff
@@ -469,8 +469,8 @@ def alert_actions(actions)
     end
   end
 
-  def modal_close(data_dismiss = 'modal', text = _('Close'))
```

Comment:
this wasn't actually used anywhere, and makes no sense to have anything other then `modal` here.